docs/sample_config.yaml

```diff
@@ -546,6 +546,21 @@ retention:
 #next_link_domain_whitelist: ["matrix.org"]
 
 
+# Enable experimental features in Synapse.
+#
+# Experimental features might break or be removed without a deprecation
+# period.
+#
+experimental_features:
+  # Support for Spaces (MSC1772), it enables the following:
+  #
+  # * The Spaces Summary API (MSC2946).
+  # * Restricting room membership based on space membership (MSC3083).
+  #
+  # Uncomment to disable support for Spaces.
+  #spaces_enabled: False
```

Comment:
This ended up higher up in the config file than might make sense? But the file is so big it is really only usable via searching anyway. So maybe this doesn't matter.

Reply:
You can reorder where it appears by moving `ExperimentalConfig` around in this list:

https://github.com/matrix-org/synapse/blob/683d6f75af0e941e9ab3bc0a985aa6ed5cc7a238/synapse/config/homeserver.py#L58-L97

It probably should go somewhere on the bottom, but at the same time it probably doesn't matter too much.

Reply:
I think it was originally this high up since it was used by `ApiConfig`, but that's no longer true (as of #9915). I'll give it a try!